app/assets/javascripts/components/features/notifications/components/notification.jsx

```diff
@@ -21,7 +21,7 @@ const Notification = React.createClass({
 
   renderFollow (account, link) {
     return (
-      <div className='notification'>
+      <div className='notification notification--follow'>
```

Comment:
When it's a namespaced (nested) element: `parent__child`. When it's a modifier: `parent-modifier`. I see nothing wrong with this PR in general but the format of these needs to be fixed.